docs/objetivo-4.md

```diff
@@ -0,0 +1,13 @@
+# Documentación hito 4
+
+## Elección del gestor de tests: pytest
+
+Las posibles opciones eran Robot(el más popular), PyTest, UnitTest y DocTest. Robot a pesar de ser muy completo tenía demasiadas funcionalidades que no se le darán uso en este proyecto y DocTest presenta una sintaxis muy fea que no me acaba de convencer. Entre UnitTest y PyTest elegí PyTest debido a que tiene una mayor popularidad y está muy depurado.

```

Comment:
Unittest y Pytest no son comparables. Enlaza de lo que estás hablando. Y tratad de entender lo que os pido, no os limitéis a copiar lo que dice no sé qué artículo.